src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/MigrateVisitRequestDto.kt

```diff
@@ -53,4 +53,6 @@ data class MigrateVisitRequestDto(
   val visitors: Set<@Valid VisitorDto>? = setOf(),
   @Schema(description = "Visit notes", required = false)
   val visitNotes: Set<@Valid VisitNoteDto>? = setOf(),
+  @Schema(description = "Username for user who actioned this request", required = false)
+  val actionedBy: String? = null,
```

Comment:
@lbennett-moj do you have this information, If you do would like to change this to required = true

Reply:
Having spoken to @lbennett-moj she did mention this information would be available. However we could leave this nullable till the story is implemented on their end.

Reply:
it will need a change in our prisoner-api service to retrieve the NOMIS username - which I think Richard has added a trello ticket for. So maybe nullable for now would be good

Reply:
unless, we leave switching to this endpoint until the username work has been done